test/controllers/api2_controller_test.rb

```diff
@@ -141,7 +147,7 @@ def test_post_minimal_observation
     params = { api_key: api_keys(:rolfs_api_key).key, location: "Earth" }
     post(:observations, params: params)
     assert_no_api_errors
-    obs = Observation.last
+    obs = Observation.unscoped.last
```

Comment:
I think this is dangerous.  If this eliminates the `ORDER BY` then you are subject to whatever ordering MySQL decides to use which changes depending on the amount of data in the given database table.  Specifically you are not guaranteed to get the Observation with the highest `id`.  If that is intended, then that should be done explicitly.  I'm sure we've been fast and lose about this in our tests, but if you're going to be explicit about it, then I would be correctly explicit about it.